lib/saluki-components/Cargo.toml

```diff
@@ -7,6 +7,7 @@ repository = { workspace = true }
 
 [dependencies]
 ahash = { workspace = true }
+arc-swap = { workspace = true }
```

Comment:
Shouldn't need to add this here since you're not actually using it in the crate.